api/src/pcapi/core/educational/models.py

```diff
@@ -957,6 +957,15 @@ def is_cancellable_from_offerer(self) -> bool:
             CollectiveBookingStatus.CANCELLED,
         )
 
+    def is_status_confirmed(self) -> bool:
+        return self.status == CollectiveBookingStatus.CONFIRMED
+
+    def is_status_cancelled(self) -> bool:
+        return self.status == CollectiveBookingStatus.CANCELLED
+
+    def is__status_reimbursed(self) -> bool:
```

Comment:
Typo, il y a 2 underscores consécutifs.

(Mon avis non sollicité : Je ne suis pas tout à fait convaincu de l'intérêt de ces fonctions, qui pourraient d'ailleurs être des propriétés.)

Reply:
Merci pour ton commentaire Damien :)

Concernant les fonctions en question d'apres Martin Fowler je cite : 

"If you have to spend effort looking at a fragment of code and figuring out what it’s doing, then you should extract it into a function and name the function after the “what.” Then, when you read it again, the purpose of the function leaps right out at you, and most of the time you won’t need to care about how the function fulfills its purpose (which is the body of the function)."

il dit aussi qu'avec un ensemble des autres methodes de refactorisation ( que j'avoue ne pas encore maitrisé a mon niveau bien que déterminé a acquerir cette compétence, la fonction peut etre lisible dans son ensemble beaucoup plus clairement )
désolé, je trouve plus la page de cette derniere citation :melting_face:

Cela étant comme précédemment décrit, je m'entraine et je propose :) au pire j'apprend :D 

Ha oui pour les property et bien, le livre de martin fowler est en javascript et j'ai due plus ou moins transformer en python du coup... J'en sais rien, je te fais confiance :D ( cela dis je te fais aussi confiance pour le reste hein ? )